.drone.star

```diff
@@ -216,9 +226,9 @@ def testing(ctx, coreBranch = 'master', coreCommit = '', phoenixBranch = 'master
           'git clone -b %s --single-branch --no-tags https://github.com/owncloud/phoenix.git /srv/app/phoenix' % (phoenixBranch),
           'cp -r /srv/app/phoenix/tests/acceptance/filesForUpload/* /uploads',
           'cd /srv/app/phoenix',
-		] + ([
+        ] + ([
           'git checkout %s' % (phoenixCommit)
-		] if phoenixCommit != '' else []) + [
+        ] if phoenixCommit != '' else []) + [
```

Comment:
indent look strange

Reply:
you were using tabs,
i am using spaces now like in the restt of the file?